frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/OutTag.java

```diff
@@ -86,6 +88,11 @@ protected String escapeInSingleQuote(String value) {
         return value.replaceAll("'", quoteReplacement("\\\'"));
     }
 
+    private String prepareTitleAttribute(String value) {
+        // Escape attribute and replace line breaks in span title element otherwise a syntax error occurs
+        return escapeAttributeValue(value).replaceAll("[\r\n]+", "&#013;&#010;");
```

Comment:
speaking of escaping... I'm not sure this replaceAll is adequate. Looks like a string "\n\r\n" will be converted to "&#013;&#010;&#013;&#010;&#013;&#010;"

Reply:
Added missing ´sw360:out´ for:
- component name (view)
- release name (clearing status)
- linked releases (vendor, name, version)